src/screens/tabs/Profile/screens/Settings/index.tsx

```diff
@@ -154,10 +161,11 @@ function Settings({ navigation }: ScreenProps<Routes.SETTINGS>) {
           </Text>
         </View>
       </ScrollView>
-      <DeleteWallet modalRef={deleteWalletRef} onDelete={handleDeleteWallet} />
+      <ExportPem modalRef={exportPemRef} />
```

Comment:
each modal we add here is a piece of my heart breaking

Reply:
:KEKW: Totally bro, don't worry, we will find out the best approach soon!